src/classes/frDonorTest.cls

```diff
@@ -56,8 +56,13 @@ public class frDonorTest {
         Test.stopTest();
         
         String frId = String.valueOf(getTestRequest().get('id'));
-        Integer newContact = [SELECT COUNT() FROM Contact WHERE fr_Id__c = :frId];
-        System.assertEquals(1, newContact, 'The funraise donor id was not populated to the contact field');
+        if(!frUtil.hasNPCobjects()) {
+            Integer newContact = [SELECT COUNT() FROM Contact WHERE fr_Id__c = :frId];
+            System.assertEquals(1, newContact, 'The funraise donor id was not populated to the contact field');
+        }
+        else {
+            Integer newContact = [SELECT COUNT() FROM Account];
```

Comment:
If there isn't anything being asserted from this, do we nee dthis change?  It seems like this test should be asserting that the donor was created as an Account/PersonAccount

Reply:
This ill work on updating, but shouldn't affect the tests or anything timeline wise. LMK if this should be prioritized!